README.md

```diff
@@ -2,6 +2,116 @@
 Service for managing jobs in the microdata platform.
 
 
+## Endpoints
+
+> ### **[GET]** `/jobs`
+>Query for existing jobs.
+><details>
+>  <summary>Example request</summary>
+>  
+>  ```curl <url>/jobs?status=queued&operation=ADD,REMOVE```
+></details>
+><details>
+>  <summary>Responses</summary>
+>
+>  | status | json                                   |
+>  |--------|----------------------------------------|
+>  |   200  |```[{...}, {...}]```                    |
+>  |   400  |```{"message": "<Error message>"}```    |
+>  |   500  |```{"message": "<Error message>"}```    |
+>
+></details>
+><details>
+>  <summary>Query Parameters</summary>
+>
+>  * **status** - filter on job status
+>  * **operation[]** - filter on job operation
+>  * **ignoreCompleted** - ignore completed jobs True | False
+></details>
+_____
+> ### **[GET]** `/jobs/<jobId>`
+>Get one existing job by job id
+><details>
+>  <summary>Example request</summary>
+>  
+>  ```curl <url>/jobs/123```
+></details>
+><details>
+>  <summary>Responses</summary>
+>
+>  | status | json                                   |
+>  |--------|----------------------------------------|
+>  |   200  |```{"message": "OK"}```                 |
+>  |   400  |```{"message": "<Error message>"}```    |
+>  |   500  |```{"message": "<Error message>"}```    |
+>
+></details>
+_____
+> ### **[POST]** `/jobs`
+>Post new jobs
+><details>
+>  <summary>Example request</summary>
+>  
+>  ```curl -X POST <url>/jobs -d '{"jobs": [{...}, {...}]}'```
+></details>
+></details>
+><details>
+>  <summary>Responses</summary>
+>
+>  | status | json                                                                                        |
+>  |--------|---------------------------------------------------------------------------------------------|
+>  |   200  |```[{"status": "CREATED", "msg": "OK"}, {"status": "FAILED", "msg": "Missing operation"}]``` |
+>  |   400  |```{"message": "<Error message>"}```                                                         |
+>  |   500  |```{"message": "<Error message>"}```                                                         |
+>
+></details>
+_____
+> ### **[PUT]** `/jobs`
```

Comment:
PUT /job?

Reply:
Good catch! I fixed the endpoint url now 👍🏻  it should be "/jobs/{jobid}" to be as RESTful as possible